.github/workflows/linux.yml

```diff
@@ -132,99 +119,157 @@ jobs:
         run: |
           mkdir -p $CCACHE_DIR
 
-      - name: Get number of CPU cores
-        uses: SimenB/github-actions-cpu-cores@v2
-        id: cpu-cores
-
-      - name: CMake configure
+      - name: CMake configure - OpenVINO
         run: |
           cmake \
-            -GNinja \
+            -G "${{ env.CMAKE_GENERATOR }}" \
             -DENABLE_CPPLINT=OFF \
             -DENABLE_NCC_STYLE=OFF \
             -DENABLE_TESTS=ON \
-            -DENABLE_PYTHON=ON \
-            -DCMAKE_VERBOSE_MAKEFILE=ON \
-            -DCMAKE_BUILD_TYPE=Release \
-            -DBUILD_SHARED_LIBS=ON \
-            -DENABLE_ONEDNN_FOR_GPU=OFF \
-            -DENABLE_OV_ONNX_FRONTEND=ON \
-            -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF \
             -DENABLE_STRICT_DEPENDENCIES=OFF \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-            -DENABLE_SYSTEM_SNAPPY=ON \
             -DENABLE_SYSTEM_TBB=ON \
+            -DENABLE_SYSTEM_OPENCL=ON \
+            -DENABLE_SYSTEM_PUGIXML=ON \
             -DBUILD_nvidia_plugin=OFF \
-            -DENABLE_DEBUG_CAPS=ON \
             -DCUSTOM_OPERATIONS="calculate_grid;complex_mul;fft;grid_sample;sparse_conv;sparse_conv_transpose" \
             -DOPENVINO_EXTRA_MODULES=${OPENVINO_CONTRIB_REPO}/modules \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }} \
+            -DCMAKE_COMPILE_WARNING_AS_ERROR=ON \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=${{ env.CMAKE_CXX_COMPILER_LAUNCHER }} \
+            -DCMAKE_C_COMPILER_LAUNCHER=${{ env.CMAKE_C_COMPILER_LAUNCHER }} \
             -S ${OPENVINO_REPO} \
             -B ${BUILD_DIR}
 
       - name: Clean ccache stats
         run: ccache --zero-stats --show-config
 
-      - name: Build
-        run: cmake --build ${BUILD_DIR} --parallel ${{ steps.cpu-cores.outputs.count }} --config Release
+      - name: Cmake build - OpenVINO
+        run: cmake --build ${BUILD_DIR} --parallel --config ${{ env.CMAKE_BUILD_TYPE }}
 
-      - name: Show ccache stats
-        run: ccache --show-stats
+      - name: Cmake & Build - OpenVINO Contrib
+        if: ${{ 'false' }}
```

Comment:
Is this intentional?

Reply:
In next PR I want to split Contrib build from main OpenVINO build to avoid possible artifacts from OpenVINO contrib in `openvino.tar.gz` package

Currently, it's kind of dummy step, which can be removed (but if CI pass in product-config, I want to merge with this change and rework in next PR)

Reply:
Continued here https://github.com/openvinotoolkit/openvino/pull/20033